.gitignore

```diff
@@ -1 +1,2 @@
-.lovelyignore 
+.vscode
```

Comment:
This *should* go in `.git/info/exclude`, but since 98% of Balatro players can't read, 99.8% of them don't know what that is. I'm okay keeping this.